logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/creating/filesystem/hints/hints.kt

```diff
@@ -35,16 +36,30 @@ inline fun <T> Descriptive.DescriptionOption<Descriptive.FinalStep>.withIOExcept
 
 fun Descriptive.DescriptionOption<Descriptive.FinalStep>.withFileAttributesFailureHint(
     expect: Expect<IoResult<BasicFileAttributes>>
+): Descriptive.DescriptionOption<DescriptiveAssertionWithFailureHint.FinalStep> =
+    withAttributeFailureHint(expect) { it.value.fileType }
+
+fun Descriptive.DescriptionOption<Descriptive.FinalStep>.withPathAttributesFailureHint(
+    expect: Expect<IoResult<BasicFileAttributes>>
+): Descriptive.DescriptionOption<DescriptiveAssertionWithFailureHint.FinalStep> =
+    withAttributeFailureHint(expect) { it.path.type }
+
+private fun Descriptive.DescriptionOption<Descriptive.FinalStep>.withAttributeFailureHint(
```

Comment:
```suggestion
private inline fun Descriptive.DescriptionOption<Descriptive.FinalStep>.withAttributeFailureHint(
```

Inlining makes sense here, since we can avoid creating the lambda